libraries/ui-material/Accordion/style.js

```diff
@@ -0,0 +1,13 @@
+import { css } from 'glamor';
+import { themeConfig } from '@shopgate/pwa-common/helpers/config';
```

Comment:
Please don't do this. If we want the styles to be adjustable based on theme variables then the component should accept a prop to change the style.

This is creating an unnecessary dependency.